.github/workflows/checks.yml

```diff
@@ -11,11 +11,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: ['2.3', '2.4', '2.5', '2.6', '2.7', '3.0']
+        ruby: ['2.3', '2.4', '2.5', '2.6', '2.7', '3.0', '3.1', '3.2']
         exclude:
           - ruby: '2.3' # there's an additional RuboCop warning Layout/TrailingEmptyLines for some reason
+      fail-fast: false
```

Comment:
What's the significance of adding this? The commit message isn't very informative.

Reply:
fail-fast: false means that all the jobs run, even if one or more of them fail.  That allows you to debug failures on individual jobs.